examples/basic-multi-streams.html

```diff
@@ -0,0 +1,121 @@
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Multi Streams Example — Networked-Aframe</title>
+    <meta name="description" content="Dev Example — Networked-Aframe">
+
+    <script src="https://aframe.io/releases/1.2.0/aframe.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/socket.io/2.3.0/socket.io.slim.js"></script>
+    <script src="/easyrtc/easyrtc.js"></script>
+    <script src="/dist/networked-aframe.js"></script>
+
+    <script src="https://unpkg.com/aframe-randomizer-components@^3.0.1/dist/aframe-randomizer-components.min.js"></script>
+    <!--<script src="https://unpkg.com/aframe-particle-system-component@1.0.5/dist/aframe-particle-system-component.min.js"></script>-->
+    <script src="https://cdn.jsdelivr.net/gh/oneWaveAdrian/aframe-particle-system-component@aframe-1.2.0-upgrade/dist/aframe-particle-system-component.min.js"></script>
+    <script src="/js/spawn-in-circle.component.js"></script>
+  </head>
+  <body>
+    <a-scene networked-scene="
+      room: basic-multi-stream;
+      debug: true;
+      adapter: easyrtc;
+      audio: false;
+      video: true;
+    ">
+      <a-assets>
+
+        <img id="grid" src="https://img.gs/bbdkhfbzkk/stretch/https://i.imgur.com/25P1geh.png" crossorigin="anonymous">
+        <img id="sky" src="https://i.imgur.com/WqlqEkq.jpg" crossorigin="anonymous" />
+
+        <!-- Templates -->
+
+        <!-- Avatar -->
+        <template id="avatar-template">
+          <a-entity class="avatar">
+             <a-plane  color="#FFF" width="4" height="3" position="0 .6 0"  material="side: front" networked-video-source></a-plane>
```

Comment:
Can you keep the `side: back` for the webcam as well like we had for basic-video.html?
```
             <a-plane  color="#FFF" width="4" height="3" position="0 .6 0"  material="side: back" networked-video-source></a-plane>
```
It's hard to see if there is another person in the room otherwise when testing. On one of my test, I thought the second participant wasn't there but it's just I was behind him. :)

Reply:
The camera was actually really rendered behind the avatar. This is why I had a hard time testing this. :)
I fixed it in https://github.com/networked-aframe/networked-aframe/commit/38bd9773e7ce973091ffcd3850eb699a22ecc0da